guides/gatherers.md

```diff
@@ -14,32 +14,49 @@ Facts Gathering process in a nutshell
 fact = gatherer(argument)
 ```
 
+## A note of gatherers versioning
+
+When a gatherer includes a breaking change (e.g. the Rhai output format changes), a new version of that same gatherer will be introduced with a new
```

Comment:
Maybe we could use present tense  like suggested in the [suse styleguide for documentation](https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-tense). 
So instead of `will be introduced`  use maybe `is introduced`
or what do you think ? :D